src/ec/packages/ec-component-select/README.md

```diff
@@ -18,6 +18,10 @@ npm install --save @ecl-twig/ec-component-select
 - **"name"** (string) (default: '')
 - **"width"** (string) (default: '') Input width size (s, m or l)
```

Comment:
For large size, the search input is nit full-width, strange?
Maybe we should create a ECL ticket for this, what do you think?

<img width="448" alt="Screenshot 2020-07-07 at 14 18 34" src="https://user-images.githubusercontent.com/1289480/86780917-930fd000-c05d-11ea-958e-d9e21c99b931.png">

Reply:
again, @papegaill , this is just the twig implementation, this concern was already communicated about the ECL component, we decided to go this way because it was already approved by comm.

Reply:
ECL ticket created 👍